src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 40 * days;
+
+  if (days >= 7) {
+    sum -= 50;
+  } else if (days >= 3) {
+    sum -= 20;
+  }
```

Comment:
1. just use if, instead of else if, since you have early return
2. consider using a constant for the discount value 30 instead of using the hardcoded value.
3. you can just use early return, like return rentalCost - basicDiscount